app.go

```diff
@@ -41,7 +42,7 @@ type ReadOnlyState interface {
 	Config() *cctypes.Config
 	EnabledFeatures() []string
 	GetMinBuildNumber() uint64
-	GetTrieDB() *trie.Database
+	GetEVMStateDB() gstate.Database
```

Comment:
Just how we don't expose the IAVL store via ReadOnlyState/State we shouldn't be exposing the low level EVM DB.

Reply:
K, I will move it to `state`. This is not a low level database. EVMStateDB is a wrapper of loomState -> EthDB -> TrieDB -> EVMStateDB.